crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -761,6 +764,21 @@ impl<T: WasmInstance> WasmModuleInstance<T> {
             }
         }
 
+        // Multi-column indexes cannot be unique (yet), so just add them.
```

Comment:
Actually, we might be able to support it with an additional attribute on the macro. I can add it if we want it right now.

Reply:
Should that PR be split into smaller pieces maybe?